proto/interchain_security/ccv/provider/v1/genesis.proto

```diff
@@ -78,6 +78,8 @@ message ConsumerState {
   repeated interchain_security.ccv.v1.ValidatorSetChangePacketData
       pending_valset_changes = 6 [ (gogoproto.nullable) = false ];
   repeated string slash_downtime_ack = 7;
+  // the phase of the consumer chain
+  ConsumerPhase phase = 9;
```

Comment:
Why is this added here? Making the genesis export / import work requires more fields, so probably it makes sense to do it in its own PR.

Reply:
Was added because genesis work was failing. I agree that more things need to be added but if I didn't add this here, we would have tests failing and I wanted to this PR to be coherent (test-wise).